src/views/acquirer/format.njk

```diff
@@ -3,6 +3,10 @@
 {% from "govuk/components/button/macro.njk" import govukButton %}
 {% from "govuk/components/input/macro.njk" import govukInput %}
 
+{% if errorMessage %}
```

Comment:
This is repeated, we should be able to use nunjucks to deal with this across all the templates